language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -577,9 +577,11 @@ protected LSCompletionItem getImplicitNewCItemForClass(ClassSymbol symbol, Balle
         return completionItems;
     }
 
-    private CompletionItem getModuleCompletionItem(String label, String insertText, @Nonnull List<TextEdit> txtEdits) {
+    private CompletionItem getModuleCompletionItem(String label, String insertText, @Nonnull List<TextEdit> txtEdits, 
+                                                   String prefix) {
         CompletionItem moduleCompletionItem = new CompletionItem();
         moduleCompletionItem.setLabel(label);
+        moduleCompletionItem.setFilterText((prefix == null) ? "" : prefix);
```

Comment:
If prefix is null, let's avoid setting it. Then the client will use the label instead.